octobot_trading/modes/abstract_trading_mode.py

```diff
@@ -244,6 +400,40 @@ def load_config(self) -> None:
         if not self.trading_config:
             self.set_default_config()
 
+    LIGHT_VOLUME_WEIGHT = "light_weight_volume_multiplier"
+    MEDIUM_VOLUME_WEIGHT = "medium_weight_volume_multiplier"
+    HEAVY_VOLUME_WEIGHT = "heavy_weight_volume_multiplier"
+    VOLUME_WEIGH_TO_VOLUME_PERCENT = {}
+
+    LIGHT_PRICE_WEIGHT = "light_weight_price_multiplier"
+    MEDIUM_PRICE_WEIGHT = "medium_weight_price_multiplier"
+    HEAVY_PRICE_WEIGHT = "heavy_weight_price_multiplier"
+
+    async def reload_config(self, bot_id: str) -> None:
+        """
+        Try to load TradingMode tentacle config.
+        Calls set_default_config() if the tentacle config is empty
+        """
+        self.trading_config = tentacles_manager_api.get_tentacle_config(self.exchange_manager.tentacles_setup_config,
+                                                                        self.__class__)
+        # set default config if nothing found
+        if not self.trading_config:
+            self.set_default_config()
+        await self.load_and_save_user_inputs(bot_id)
+        for element in self.consumers + self.producers:
+            if isinstance(element, (abstract_mode_consumer.AbstractTradingModeConsumer,
+                                    abstract_mode_producer.AbstractTradingModeProducer)):
+                element.reload_config()
+
+    def get_local_config(self):
+        return self.trading_config
+
+    @classmethod
+    def create_local_instance(cls, config, tentacles_setup_config, tentacle_config):
+        return modes_factory.create_temporary_trading_mode_with_local_config(
+            cls, config, tentacle_config
+        )
```

Comment:
Where is it used?

Reply:
in `AbstractTentacle` when we want user inputs from a tentacle that is not running at in the current bot:
![image](https://user-images.githubusercontent.com/9078616/193407079-e882684a-5ccc-4017-97cf-d987b335d1ed.png)

Reply:
Right thanks 👍